### PR TITLE
[Cpp Graph]fix q8 pern QKV fusion of vnni

### PR DESCRIPTION
**Pull request by luoyu-intel**
## Type of Change

fix the crash of vnni 

## How has this PR been tested?

llama-7b

### CODE REVIEW

File: intel_extension_for_transformers/llm/runtime/graph/core/layers/ip_fusion_ffn.cpp [hunk 1]
```diff
@@ -54,50 +58,148 @@ bool jblas_fusion_FFN_SiLu_f32f32_support(void* w1ptr, void* w2ptr, void* w3ptr,
...
+      auto offset = workspace == NULL ? 0 : finter.getActivationPtr()->getWorkSpaceSize(seq, fin, w1ptr->mBlockSize);
+      auto quanA2 = finter.getActivationPtr()->createStorage(seq, fmid, w2ptr->mBlockSize, (int8_t*)workspace + offset);
+      ret = finter.compute({seq,   fin,   fmid, fout,   activation, lda, quanA1, tmp1, ldtmp1, quanA2, w1ptr,
+                            w2ptr, w3ptr, tmp1, ldtmp1, output,     ldo, NULL,   tmp2, ldtmp2, NULL});
+      delete quanA1;
+      delete quanA2;
+    }
   }
-  return JblasNotSupport;
+  return ret;
```
Comment (airMeng) [inline intel_extension_for_transformers/llm/runtime/graph/core/layers/ip_fusion_ffn.cpp:103]: if ```w1ptr->mBlockSize % 128 != 0 && w1ptr->mBlockSize % 4 != 0``` should throw an explicit error saying "not supported blocksize"?

File: intel_extension_for_transformers/llm/runtime/graph/core/layers/jblas_common.hpp [hunk 2]
```diff
@@ -128,24 +128,58 @@ class Silu {
...
+template <JBLAS_ISA ISA_T>
+class DequantSiluFp32 : protected jblas::epilogue::gemm::DequantInt32ToFp32<ISA_T> {
+ public:
+  using Parent = jblas::epilogue::gemm::DequantInt32ToFp32<ISA_T>;
+  using Param = typename Parent::Param;
+  using SiluKernel = jblas::epilogue::gemm::CustomAccumulatorWriteBackWithEltop<ISA_T, float, float, SWISH>;
+
+  JBLAS_CODE forward(const int32_t* cacheptr, const int cachestep, const int M_offset, const int N_offset, const int M,
+                     const int N, const Param& _param) {
+    float alpha = 1.f;
```
Comment (zhewang1-intc) [inline intel_extension_for_transformers/llm/runtime/graph/core/layers/jblas_common.hpp:152]: -1.f in swish?

File: intel_extension_for_transformers/llm/runtime/graph/core/layers/ip_fusion_ffn.cpp (hunk 1 above)
Comment (luoyu-intel) [reply intel_extension_for_transformers/llm/runtime/graph/core/layers/ip_fusion_ffn.cpp:103]: it's better to throw at quant time, not runtime.

File: intel_extension_for_transformers/llm/runtime/graph/core/layers/jblas_common.hpp (hunk 2 above)
Comment (luoyu-intel) [reply intel_extension_for_transformers/llm/runtime/graph/core/layers/jblas_common.hpp:152]: fixed

File: intel_extension_for_transformers/llm/runtime/graph/core/layers/ip_fusion_ffn.cpp (hunk 1 above)
Comment (luoyu-intel) [reply intel_extension_for_transformers/llm/runtime/graph/core/layers/ip_fusion_ffn.cpp:103]: it's not correct to throw at quant time. add one assert here should be okay.

**Review by airMeng — APPROVED** [review]

**Review by zhewang1-intc — APPROVED** [review]
LGTM

**Pull request merged**